dev/NumberBox/NumberBox.cpp

```diff
@@ -392,6 +399,10 @@ void NumberBox::OnNumberBoxGotFocus(winrt::IInspectable const& sender, winrt::Ro
             popup.IsOpen(true);
         }
     }
+    else if (SpinButtonPlacementMode() == winrt::NumberBoxSpinButtonPlacementMode::Inline)
+    {
+        //UpdateSpinButtonPlacement();
+    }
```

Comment:
I think we can remove this empty case can we?

Reply:
oops, missed that!